conformance_pack/vpc.sp

```diff
@@ -373,6 +373,36 @@ control "vpc_security_group_allows_ingress_to_oracle_ports" {
   })
 }
 
+control "vpc_in_more_than_one_region" {
+  title       = "VPCs should exist in multiple regions"
+  description = "This control checks whether there are VPCs present in multiple regions."
+  query       = query.vpc_in_more_than_one_region
+
+  tags = merge(local.conformance_pack_vpc_common_tags, {
+    other_checks = "true"
+  })
+}
+
+control "vpc_subnet_multi_az_enabled" {
```

Comment:
@khushboo9024 this should compliment to `vpc_in_more_than_one_region`, we may think to remove `vpc_in_more_than_one_region`